src/main/java/com/onelogin/saml/Utils.java

```diff
@@ -53,10 +75,12 @@ public String getNamespaceURI(String prefix) {
 				String result = null;
 				if (prefix.equals("samlp"))
```

Comment:
Sometime, IdPs uses samlp2 and saml2 as prefixes, we should be compatible with that fact.